gymnasium/utils/play.py

```diff
@@ -6,6 +6,7 @@
 from typing import Callable, List
 
 import numpy as np
+from matplotlib.axes import Axes
```

Comment:
This should be in `if TYPE_CHECKING` as I believe this will require users to install matplotlib as its imported in `import gymnasium`